### PR TITLE
Update HedgeDoc guide

### PR DESCRIPTION
**Pull request by ZPain8464**
Updates the copy, adds certificates steps and how to add a user, and update the Docker pull URL to `cr.pomerium.com/pomerium/pomerium:latest`

### CODE REVIEW

File: content/docs/guides/hedgedoc.md [hunk 1]
```diff
@@ -1,147 +1,113 @@
...
-  ]
-description: This guide covers how to use Pomerium to authenticate and authorize users of HedgeDoc.
+keywords: [pomerium, HedgeDoc, authentication, authorization]
+description: Learn how to control access to your HedgeDoc web application behind Pomerium.
 ---
 
 # HedgeDoc
 
-Use Pomerium to secure access to HedgeDoc by providing identity and context.
+In this guide, you'll learn how to run a HedgeDoc web application behind Pomerium. You'll also add an authorization policy that enables Pomerium to gran or deny access based on policy criteria.
```
Comment (cmo-pomerium) [inline content/docs/guides/hedgedoc.md:13]:
```suggestion
In this guide, you'll learn how to run a HedgeDoc web application behind Pomerium. You'll also add an authorization policy that enables Pomerium to grant or deny access based on policy criteria.
```

spelling

File: content/docs/guides/hedgedoc.md [hunk 2]
```diff
@@ -1,147 +1,113 @@
...
-- A [pre-configured IdP](https://www.pomerium.com/docs/identity-providers)
-
-:::tip
+- [mkcert](https://github.com/FiloSottile/mkcert#installation)
 
-**Note**
+:::note
 
-This guide uses [GitHub](https://www.pomerium.com/docs/identity-providers/github) as the pre-configured IdP.
+If you completed our [**Quickstart guide**](/docs/quickstart), you should have a working Pomerium project with the following YAML files:
```
Comment (cmo-pomerium) [inline content/docs/guides/hedgedoc.md:35]: Feels like I'm missing something here. 
"...the following YAML files:" with a colon but then there's nothing after it. Please check.

**Review by cmo-pomerium — CHANGES_REQUESTED** [review]
Fixed a typo and then wanted to make sure a colon was intended with nothing left out.

File: content/docs/guides/hedgedoc.md (hunk 2 above)
Comment (ZPain8464) [reply content/docs/guides/hedgedoc.md:35]: The YAML files are listed below as bullet points:
SCR-20240227-lqlj

**Review by cmo-pomerium — APPROVED** [review]
LGTM

**Pull request merged**